semantic_router/encoders/openai.py

```diff
@@ -20,17 +20,19 @@ def __init__(
         self,
         name: Optional[str] = None,
         openai_api_key: Optional[str] = None,
+        openai_org_id: Optional[str] = None,
         score_threshold: float = 0.82,
         dimensions: Union[int, NotGiven] = NotGiven(),
     ):
         if name is None:
             name = os.getenv("OPENAI_MODEL_NAME", "text-embedding-ada-002")
         super().__init__(name=name, score_threshold=score_threshold)
         api_key = openai_api_key or os.getenv("OPENAI_API_KEY")
-        if api_key is None:
+        org_id = openai_org_id or os.getenv("OPENAI_ORGANIZATION")
+        if (api_key is None) and (org_id is None):
```

Comment:
We probably want to remove `and (org_id is None)`. If `api_key is None` the API call should not run, regardless of the value of `org_id`. I think it's also okay to make the API call if `org_id` is not set (I'm fairly certain it's optional), and so I wouldn't use 'or' here either. I think `and (org_id is None)` can just be remoed.